lib/error-boundary.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class ErrorBoundary extends React.Component {
+  static propTypes = {
+    children: PropTypes.any,
```

Comment:
```suggestion
    children: PropTypes.node.isRequired,
```

`PropTypes.node` specifies "anything React can render," just to tighten it up a bit.